webapp/src/components/backstage/playbook_list.tsx

```diff
@@ -321,6 +346,17 @@ const ImportButton = (props: {onClick?: () => void}) => {
     );
 };
 
+const ImportLinkButton = (props: {onClick?: () => void}) => {
+    return (
+        <ImportLink
+            href='#'
+            onClick={props.onClick}
+        >
+            <FormattedMessage defaultMessage='Import a playbook'/>
+        </ImportLink>
+    );
+};
```

Comment:
The `#` href isn't necessary IMO; it allows open-in-new-tab flows which doesn't really make sense for this. (And the `children` to unify the message.)

```suggestion
const ImportLinkButton = (props: {children: React.ReactNode | React.ReactNode[]; onClick?: () => void}) => {
    return (
        <ImportLink onClick={props.onClick}>
            {props.children}
        </ImportLink>
    );
};
```